### ISSUE TITLE
Question about configs and skewness hyperparameters

### ISSUE THREAD

**Issue by hangg7** [Mumeishi514/d_2nerf#1]
Hi! Thanks for releasing this very interesting project and sharing the code! I am wondering why there are five different configs [here](https://github.com/d2nerf/d2nerf/tree/main/configs/rl)? 

It also seems like `skewness` is a sensitive hyperparameters as in the checkpoints you released, all experiments have there own skewness parameter (2.0, 2.75 and 2.875). I am wondering if it's the case.

Thanks!
Hang

**Comment by hangg7** [comment]
My bad, the descriptions about this is in supplement Section B. Thanks for spending time clarifying it! I am still interested in your comments about the hyperparameters in general about your method, it seems quite sensitive to make work.

Closing now.

**Comment by d2nerf** [comment]
Hi and thank you for being interested in our work. Yes, you are exactly right about the sensitivity of the hyperparameters, especially the skewness. Because d^2NeRF is fully self-supervised and can only rely on motion clues to decouple the scene, many things such as level of object/camera motion, frequency of object or background and accuracy of camera parameters can all affect the quality of decomposition. We applied the ray regularization (eq 10) to partially mitigate this issue, but for some difficult scenes, different hyperparameters might be needed.

In general, if you want to run on your own video, we recommend trying `configs/rl/001.gin` first as it works well on the majority of scenes. If this fails, then we would suggest trying the rest of the configurations for real-life scenes or fine-tuning them a bit.

Hope that helps!
